pkg/reconciler/git/clone.go

```diff
@@ -97,8 +97,9 @@ func (r *Cloner) CloneAndCheckout(dstPath, rev string) error {
 }
 
 func (r *Cloner) buildAuth() (transport.AuthMethod, error) {
-	if r.repo.TokenNamespace == "" {
-		return nil, nil
+	tokenNamespace := "default"
```

Comment:
Still not convinced that it's a good approach to let the reconciler do some Secret-lookups to retrieve Git credentials instead providing them directly as configuration-parameter. This couples the GIT library of the reconciler to company internal deployment agreements.

@piotrmiskiewicz  / @PK85 : can't we provide such credentials as part of the component configuration provided by KEB?

Reply:
I do not recall any specific mechanics for that. I suggest to handle it outside of this PR.